### PR TITLE
ID aligned to the center of the box

### PR DESCRIPTION
**Pull request by geeythree**
ID has been aligned to the center of square box. 

This PR fixes issue #2 

Screenshot : 
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -88,7 +88,13 @@ function App() {
   return (
     <main>
       {colorBox.map((x: IColorBox) => {
-        return <ColorBox onClick={() => onClick(x)} bg={x.color} key={x.id} />;
+        return(
+          <>
```
Comment (mankinchi) [inline src/App.tsx]: there is no need of this fragment

**Review by mankinchi — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (geeythree) [reply src/App.tsx]: @mankinchi Did you check the issue?

Comment (mankinchi) [reply src/App.tsx]: yes I have. You can simply return a ColourBox and the div in it. No need to include `<>` around the `ColorBox`

Comment (geeythree) [reply src/App.tsx]: Oh okay! Thanks for pointing it out 😄

**Review by redxzeta — APPROVED** [review]

**Pull request merged**